src/command_modules/azure-cli-cognitiveservices/azure/cli/command_modules/cognitiveservices/commands.py

```diff
@@ -4,27 +4,29 @@
 # --------------------------------------------------------------------------------------------
 
 from azure.cli.core.commands import CliCommandType
-from azure.cli.command_modules.cognitiveservices._client_factory import (
-    cf_cognitive_service_accounts,
-    cf_accounts)
+from azure.cli.command_modules.cognitiveservices._client_factory import cf_accounts, cf_resource_skus
 
 
 def load_command_table(self, _):
     mgmt_type = CliCommandType(
-        operations_tmpl='azure.mgmt.cognitiveservices.operations.cognitive_services_accounts_operations#CognitiveServicesAccountsOperations.{}',  # pylint: disable=line-too-long
-        client_factory=cf_cognitive_service_accounts
+        operations_tmpl='azure.mgmt.cognitiveservices.operations.accounts_operations#AccountsOperations.{}',
+        client_factory=cf_accounts
     )
 
     with self.command_group('cognitiveservices account', mgmt_type) as g:
         g.custom_command('create', 'create')
         g.command('delete', 'delete')
         g.show_command('show', 'get_properties')
         g.custom_command('update', 'update')
+        g.custom_command('list', 'list_resources')
         g.command('list-skus', 'list_skus')
+        g.custom_command('list-usages', 'list_usages')
+        g.custom_command('list-kinds', 'list_kinds', client_factory=cf_resource_skus)
 
     with self.command_group('cognitiveservices account keys', mgmt_type) as g:
         g.command('regenerate', 'regenerate_key')
         g.command('list', 'list_keys')
 
+    # deprecating this
     with self.command_group('cognitiveservices') as g:
-        g.custom_command('list', 'list_resources', client_factory=cf_accounts)
+        g.custom_command('list', 'list_resources', deprecate_info='az cognitiveservices account list')
```

Comment:
This should use the new style of deprecation. Up to you whether to hide it or select an expiration version. 
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#deprecating-commands-and-arguments

Reply:
One question, in the example: `deprecate_info=g.deprecate(redirect='test show', expiration='2.1.0')` 2.1.0 is the version of azure-cli or the version of azure-cli-cognitiveservices

Reply:
azure-cli version.  The safest thing would be to not put an expiration but just to hide it.